keepassxc-browser/content/ui.js

```diff
@@ -126,7 +126,8 @@ kpxcUI.updateIconPosition = function(iconClass) {
 };
 
 kpxcUI.calculateIconOffset = function(field, size) {
-    const offset = Math.floor((field.offsetHeight - size) / 3);
+    const offset = Math.floor((field.offsetHeight / 2) - (size / 2) - 1);
+    console.log(field.offsetHeight, size, offset);
```

Comment:
No need to write this to the log, I assume you left that in here from debugging.

Reply:
Totally missed that. Fixed :)